app/views/publish/index.html.erb

```diff
@@ -72,12 +80,21 @@
 
     <%= render partial: 'autocomplete_warning' %>
 
+    <button class="govuk-button fb-govuk-button"
+            data-module="govuk-button"
+            type="button" 
+            aria-disabled="<%= @autocomplete_warning.messages.blank? ? 'false' : 'true' %>"
```

Comment:
```suggestion
            aria-disabled="<%= ActiveModel::Type::Boolean.new.cast(@autocomplete_warning.messages.blank?).to_s %>"
```

Reply:
actually might not even be possible. am unsure if `ActiveModel` is available in a view

Reply:
I believe your replacement would be the wrong way round, so I'd need:
`ActiveModel::Type::Boolean.new.cast(!@autocomplete_warning.messages.blank?).to_s`
or maybe 
`ActiveModel::Type::Boolean.new.cast(@autocomplete_warning.messages.length).to_s`

Reply:
ah yeah quite right:

`ActiveModel::Type::Boolean.new.cast(@autocomplete_warning.messages.present?).to_s`

either way, don't worry :)